packages/preview/game-theoryst/0.1.0/src/utils/table_utils.typ

```diff
@@ -0,0 +1,36 @@
+#import table: cell
+#import "programming_utils.typ": if-else
+
+
+// Utility to insert blank cells
+#let blank_cells(cspn: 3) = {
```

Comment:
Typst packages generally use kebab-case names. Can you please rename this function to `blank-cells`? Kebab case is only a convention, so if you want to keep it as is, it is also possible, just tell me what you prefer.

Reply:
Updated to reflect both changes. Thank you!